packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -651,8 +651,12 @@ class VirtualListBasic extends Component {
 
 		// reset
 		this.scrollPosition = 0;
-		if (this.props.scrollMode === 'translate' && this.contentRef.current) {
+		if (scrollMode === 'translate' && this.contentRef.current) {
 			this.contentRef.current.style.transform = null;
+		} else if (scrollMode === 'native' && node) {
+			node.style.scrollBehavior = null;
+			this.updateScrollPosition(this.getXY(this.scrollPosition, 0));
+			node.style.scrollBehavior = 'smooth';
```

Comment:
How to save and restore the previous value without specifying the specific value?'

const scrollBehavior = node.style.scrollBehavior;  
..
node.style.scrollBehavior = scrollBehavior;

Reply:
I think we should clean up this pattern later.